### ISSUE TITLE
CVE-2021-33587 package vulnerability

### ISSUE THREAD

**Issue by dilumn** [mjmlio/mjml#2284]
**Describe the bug**

```
=> Found "css-what@2.1.3"
info Reasons this module exists
   - "_project_#mjml-core#cheerio#css-select" depends on it
```
https://avd.aquasec.com/nvd/cve-2021-33587/

Is it possible to update mjml packages to have `css-what` >= 5.0.1?
Looks like for this to fully work we need https://github.com/Automattic/juice package to update their dependencies also. But that can be done from `juice` side

**Expected behavior**
css-what to be `5.0.1` in yarn.lock file

**MJML environment (please complete the following information):**
 - MJML Version - latest

**Comment by pfmnzone** [comment]
According to [this](https://github.com/cheeriojs/cheerio/issues/1924#issuecomment-856883288) updating to cheerio@1.0.0-rc.10 should do the trick.

Here is a [pull request](https://github.com/mjmlio/mjml/pull/2285)

**Comment by mashpie** [comment]
please don't forget to publish :)

**Comment by mashpie** [comment]
@iRyusa Thanks for info. Is there any eta on when you will release? Because, currently, some CI Pipelines fail and block due to that high CVE. If it takes some more time, I'd rather tend to monkey patch cheerio and/or css-what though `resolutions`  in yarn than waiting too long for a release.

**Comment by iRyusa** [comment]
We'll see if we can get sometime around next week as we can't automate releases.

**Comment by iRyusa** [comment]
This should be solved in 4.10.0, note that there's a dep issue with babel runtime you should regenerate your lockfile 👍  closing

**Comment by mashpie** [comment]
@iRyusa perfect time! I can confirm all working fine. Thanks a lot!
